code/modules/psionics/standardpowers.dm

```diff
@@ -152,7 +152,7 @@
 /obj/item/organ/internal/psionic_tumor/proc/psionic_weapon()
 	set category = "Psionic powers"
 	set name = "Psionic Weapon Construction (3)"
```

Comment:
```suggestion
	set name = "Psionic Range Construction (3)"
```

Reply:
The name was too long when called "Psionic range weapon construction" and range in place of weapon is a little harder to understand versus melee and tool for the other powers.

Reply:
Fair enough